recipes/nomad-camels/meta.yaml

```
@@ -0,0 +1,125 @@
+{% set name = "nomad-camels" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nomad_camels-{{ version }}.tar.gz
+  sha256: 76c68fb99d3df97faae219f1abc501402a09fec6ec6486eec1d82f41743cd2cd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9.6
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.9.6
+    - pyside6 >=6.5.1
+    - bluesky >=1.8.3
+    - ophyd >=1.6.4
+    - databroker >=1.2.5
+    - pandas >=1.4.2
+    - numpy >=1.22.4
+    - h5py >=3.7.0
+    - xarray >=2022.3.0
+    - qtpy >=2.1.0
+    - pip >=21.1.2
+    - wheel >=0.36.2
+    - setuptools >=57.0.0
```

Are these really needed?

```suggestion
    - pip >=21.1.2
    - wheel >=0.36.2
    - setuptools >=57.0.0
```

---

That is what they require in their `requirements.txt` file https://github.com/FAU-LAP/NOMAD-CAMELS/blob/main/requirements.txt - I agree that it could run without those but I just aimed to be consistent with their `requirements.txt` file.

---

Looks like bad metadata updatream :-/